extensions/functions_datetime.yaml

```diff
@@ -48,7 +53,11 @@ scalar_functions:
             value: timestamp_tz
           - name: y
             value: interval_year
-        return: timestamp
+          - name: timezone
+            constant: true
```

Comment:
Remove.  Pending ML discussion on constant arguments in scalar functions.

Reply:
Removed.